cloud/amazon/ec2_asg.py

```diff
@@ -112,8 +112,8 @@
     version_added: "2.0"
   wait_timeout:
     description:
-      - how long before wait instances to become viable when replaced.  Used in concjunction with instance_ids option.
-    default: 300
+      - The number of seconds to wait for instances to become healthy after they are replaced.  Used in conjunction with the replace_instances option.
+    default: 300 seconds
```

Comment:
agreed, remove 'seconds' from default: entry, it already states the unit in the description